doc/index.rst

```diff
@@ -0,0 +1,48 @@
+Welcome to Marian's documentation!
+==================================
+
+|buildgpu| |buildcpu| |tests| |release| |license|
+
+Marian is an efficient and self-contained Neural Machine Translation framework with an integrated
+automatic differentiation engine based on dynamic computation graphs, written entirely in C++.
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   install
+   graph
+   operators
+
+   api/library_index
+
+   contributing
+
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
```

Comment:
I guess the links of `modindex` and `search` have not been set yet.  The `Module Index` page cannot be accessed. The search function does not work.

Reply:
Good catch! I disabled `modindex` as it isn't very relevant to C++ docs.
Search had two issues:
1. Searching didn't work in Sphinx 2.0.0, so I updated it to highest 2.x.x, which currently is 2.4.4.
2. The search page is empty by default in the `sphinx_rtd_theme`, but there seem to be some PRs open for this.
So I simply removed `search`, because it's accessible from the top of the side bar anyway.